test/features/users.feature

```diff
@@ -146,7 +146,7 @@ Feature: User APIs
         When I request a password reset for "publisher"
         Then I should see an element with xpath "//div[contains(string(), 'A reset link has been emailed to you')]"
 
-    Scenario: Register user password must be 10 characters or longer
+    Scenario: Register user password must be 10 characters or longer and contain number, lowercase, capital, and symbol
```

Comment:
Do we have tests for lockout, non lockout but has numbrs in redis etc?

Reply:
Not currently, no.